packages/beacon-state-transition/src/allForks/util/epochContext.ts

```diff
@@ -137,7 +138,11 @@ export function createEpochContext(
         exitQueueChurn += 1;
       }
     }
+
+    // TODO: Should have 0 for not active validators to be re-usable in ForkChoice
```

Comment:
@tuyennhv I think it's unsafe to pass to the forkchoice since in case of heavy forking there may still be attestations around for attackers. We should ping the researchers before going that way

Reply:
@dapplion we could loop through the cached effectiveBalances persistent-ts 1 time anyway, will create a PR for that